doc/classes/TabContainer.xml

```diff
@@ -110,6 +114,7 @@
 			<argument index="0" name="popup" type="Node">
 			</argument>
 			<description>
+				When set to a [class Popup] node instance, a popup menu icon appears in the top-right corner of the [class TabContainer]. Clicking it will expand the [class Popup] node.
```

Comment:
"When set to a" -> "If set on a"

Reply:
Wouldn't that phrasing imply you are setting a property that belongs to a Popup node (the property is "on" the node) rather than setting the property's value "to" a Popup node?

Reply:
I made the change anyway for now.